packages/cli/package.json

```diff
@@ -30,8 +30,6 @@
   },
   "dependencies": {
     "@arethetypeswrong/cli": "^0.15.2",
-    "@osdk/generator": "workspace:~",
```

Comment:
I assume were good to remove this because its going through .cmd.typescript? If so, does .cmd.typescript have the strict dep on the generator without the ~?

Reply:
Yeah this was directly unused and transitively used by the cmd.typescript.

And since all cli osdk things are being bundled, the line in cmd.typescript is irrelevant (we dont even publish cmd.typescript)